src/server/acceptor/rustls.rs

```diff
@@ -22,10 +26,10 @@ impl RustlsAcceptor {
 }
 
 impl Acceptor for RustlsAcceptor {
-    type Future = Accept<TcpStream>;
-    type Stream = TlsStream<TcpStream>;
+    type Stream = Box<TlsStream<TcpStream>>;
```

Comment:
I'll probably find a better way to keep the size of tls streams below 2048. For now, this is fine. It's not like people are filling up the backlog with issues complaining about poor performance when tls is in use.